samples/nrf_desktop/src/modules/ble_scan.c

```diff
@@ -24,66 +22,24 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_SCANNING_LOG_LEVEL);
 #define DEVICE_NAME	CONFIG_DESKTOP_BLE_SHORT_NAME
 #define DEVICE_NAME_LEN	(sizeof(DEVICE_NAME) - 1)
 
-static bt_addr_le_t target_addr;
-
-
-static void save_address(struct bt_conn *conn)
-{
-	struct bt_conn_info info;
-
-	int err = bt_conn_get_info(conn, &info);
-
-	if (err) {
-		LOG_ERR("Cannot get conn info");
-		return;
-	}
-
-	if (bt_addr_le_cmp(&target_addr, info.le.dst)) {
-		LOG_INF("Store target address");
-		bt_addr_le_copy(&target_addr, info.le.dst);
-		settings_save_one(MODULE_NAME "/taddr", &target_addr,
-				  sizeof(target_addr));
-	}
-}
-
-static int settings_set(int argc, char **argv, void *val_ctx)
-{
-	if (argc == 1) {
-		if (!strcmp(argv[0], "taddr")) {
-			int len = settings_val_read_cb(val_ctx, &target_addr,
-						       sizeof(target_addr));
-
-			if (len < 0) {
-				LOG_ERR("Can't read target address (err:%d)",
-					len);
-			}
-		}
 
-		return 0;
-	}
+struct bond_scan_data {
```

Comment:
This struct description is duplicated in ble_adv and ble_scan

Reply:
discussed offline - let's keep this as is
We could create an additional directory for common code (e.g. `src/utils`). However at this point this is the only thing that would go there. So the triage is about either to duplicate a small function or maintain another file. We decided to duplicate for this case.